src/os_view.eliom

```diff
@@ -68,6 +86,10 @@ let%shared connect_form ?a () =
         Form.string;
     ]) ()
 
+(** [disconnect_button ~a ()] creates a disconnect POST form with a button
+    without value, a signout icon and a text message "logout".
+
+    @param a modify the attributes of the form. *)
```

Comment:
I'm not sure `modify`  is the right word. It's semantics is afaik functional, and `modify` sounds like there's some imperative assignment...